doc/api/next_api_changes/removals/22516-OG.rst

```diff
@@ -0,0 +1,20 @@
+Removal of deprecations in ``backends``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The parameter ``resize_callback`` is removed from the TK backend. Instead,
+use ``get_tk_widget().bind('<Configure>', ..., True)``.
+
+The ``get_content_extents()`` and ``tostring_rgba_minimized()`` methods are
+removed from the Agg backend with no replacements.
+
+The parameter ``dpi`` is removed from ``_print_ps()`` in the PS backend and
```

Comment:
`_print_ps` is private; it's `print_ps` and `print_eps` that have the parameter removed.